app/controllers/invites_controller.rb

```diff
@@ -195,7 +195,14 @@ def resend_invite
   def resend_all_invites
     guardian.ensure_can_resend_all_invites!(current_user)
 
-    Invite.resend_all_invites_from(current_user.id)
+    Invite
+      .left_outer_joins(:invited_users)
+      .where(invited_by: current_user)
+      .where('invites.email IS NOT NULL')
+      .where('invited_users.user_id IS NULL')
```

Comment:
```suggestion
      .where('invites.email IS NOT NULL AND invited_users.user_id IS NULL')
```

Small nit.

Reply:
It was like that before, but I prefer one condition per `.where`. :rofl: